src/screens/HomePage/HomePage.style.tsx

```diff
@@ -150,6 +152,20 @@ export const Row = styled.div`
   }
 `;
 
+export const UnderMapText = styled(Typography)`
+  ${anchorStyles};
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  font-size: 0.9rem;
+  text-align: center;
+  margin-top: 1rem;
+  max-width: 300px;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    max-width: initial;
```

Comment:
lets use `unset`

Reply:
Yea, I understand it as the "initial" css value as though it has not been explicitly set (in our case, max width of 300px). So by setting it to initial, it would inherit from the closest higher level component. I might be wrong though. We can be explicit and set it to "unset" to be sure 👍🏼